### DISCUSSION TITLE
Fooocus Performance: I need a RAM upgrade or my GPU is too old?

### DISCUSSION THREAD

**Discussion by AndreiLungu64** [lllyasviel/Fooocus#1357] [Q&A]
I am considering upgrading my laptop's RAM to either 16GB or 24GB to enhance Fooocus performance (currently, I can't run it locally). I am wondering if my current GPU, the GTX 1660 TI (6GB), and the RAM upgrade alone would suffice. Any insights are appreciated!

**Comment by lllyasviel** [comment]
GTX 1660 TI (6GB) should be able to use it no matter what ram is but will be slower than recent cards
if you cannot use it now, see also https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md

**Comment by xdgabrijuve99** [comment]
@TheFatherCat i have gtx1660ti too.  128 of its 1408 cores are dedicated FP16 cores.   I cannot use fooocus but i can use comfy ui (fp32) with sd1.5 model.  Sdxl models work with some tricks(https://youtu.be/tQK3DLJzPz0 ,i followed this video , sdxl models now work but i have black images with 1.5 models lmao)
